audio/midi_drivers/FluidSynthMidiDriver.cpp

```diff
@@ -64,13 +68,22 @@ int FluidSynthMidiDriver::open() {
 	if (!soundfont.empty())
 		soundfonts.push_back(soundfont);
 
+	_settings = new_fluid_settings();
+
 	if (soundfonts.empty()) {
-		perr << "FluidSynth requires a 'fluidsynth_soundfont' setting" << std::endl;
-		return -2;
+		char *default_soundfont;
+		if (getStr("synth.default-soundfont", &default_soundfont) == FLUID_OK &&
+		    default_soundfont && default_soundfont[0] != 0) {
+			soundfonts.push_back(default_soundfont);
+			fluid_free(default_soundfont);
```

Comment:
`fluid_free` requires FluidSynth 2.07 or higher (see [here](https://www.fluidsynth.org/api/group__misc.html#ga6f11f4e3f8d4eacf38e7e3fe884f3d7b)). This is causing the LGTM check to fail because it is not verified by the build system. Might as well use `free` directly instead.

Reply:
And Travis compiles are failing as well.